basic/tile4bpp.bas

```diff
@@ -16,23 +21,23 @@
 270 NEXT X
 280 RETURN
 
-300 REM SET UP L1
-310 VPOKE 15,$3005,$18 :REM TILES AT 24*1024
-320 VPOKE 15,$3003,$10 :REM MAP AT 16*1024
-330 VPOKE 15,$3001,$0E :REM MAPH=256 MAPW=128
-340 VPOKE 15,$3000,$61 :REM MODE 3 (4 BPP)
+300 REM SET UP GRAPHICS L0 (L1 FOR R33)
+310 VPOKE 15,GL+5,$18
+320 VPOKE 15,GL+3,$10
+330 VPOKE 15,GL+1,$0E
+340 VPOKE 15,GL+0,$61
 350 RETURN
 
-400 REM FILL PALETTE 16..31
+400 REM FILL PALETTE
 410 SLOT = $1000+32
 420 FOR X = 0 TO 15
 430 READ R,G,B
 440 VPOKE 15, SLOT+X*2, G*16+B
 450 VPOKE 15, SLOT+X*2+1, R
 460 NEXT X
-470 U=VPEEK(15,SLOT):VPOKE 15,$1000,U :REM COPY ENTRY 16 TO 0
+470 U=VPEEK(15,SLOT):VPOKE 15,$1000,U
 480 U=VPEEK(15,SLOT+1):VPOKE 15,$1001,U
-490 POKE 646,1:PRINT CHR$(147) :REM CLEAR SCREEN WITH NEW BACKGROUND
+490 POKE 646,1:PRINT CHR$(147)
```

Comment:
I think that you should write that color number as two-digit hex.  Then, it will be more obvious that the number describes both background and foreground colors.